wagtail/search/tests/test_backends.py

```diff
@@ -428,6 +428,234 @@ def test_delete(self):
             "The Fellowship of the Ring"
         ])
 
+    #
+    # Basic query classes
```

Comment:
Can we move these tests into a mixin class add only add it on to backends that support the entire API?

Reply:
Sure, I’ll do that.